elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -3,168 +3,157 @@
 <script type="module" src="rh-secondary-nav.js"></script>
 
 <div class="mock-nav"></div>
-<rh-secondary-nav role="navigation">
+<rh-secondary-nav role="navigation" data-analytics-region="secondary nav">
   <a href="#" slot="logo" id="logo-id">
     Red Hat Ansible Automation Platform
   </a>
   <ul slot="nav" aria-labelledby="logo-id">
     <li>
       <rh-secondary-nav-dropdown>
-        <a href="#" slot="link">Explore</a>
+        <a href="#" data-analytics-level="1" data-analytics-text="Explore" slot="link">Explore</a>
         <rh-secondary-nav-menu slot="menu">
           <rh-secondary-nav-menu-section>
             <h2 slot="header" id="ansible-platform">
-              <a href="#">Why Red Hat Ansible Automation Platforms</a>
+              <a href="#" data-analytics-level="2" data-analytics-text="Why Red Hat Ansible Automation Platforms" data-analytics-category="Explore">
```

Comment:
I think this category value, and others at this level, would also be *-category="Explore|Why Red Hat Ansible Automation Platforms" since it's within the same region, even though it's not inside of the <ul>

Reply:
@jvprh Are you thinking the a tag inside the h2 tag should include its own text as part of its data-analytics category? Based on the main nav example and talking with Allen that didn't seem needed.